src/Microsoft.AspNetCore.Mvc.RazorPages/Resources.resx

```diff
@@ -141,4 +141,7 @@
   <data name="PathMustBeAnAppRelativePath" xml:space="preserve">
     <value>Path must be an application relative path that starts with a forward slash '/'.</value>
   </data>
+  <data name="NoRoutesMatched" xml:space="preserve">
+    <value>No route matches the supplied values.</value>
+  </data>
```

Comment:
Should this say `no page matches`? Should this include what the route values were?